### DISCUSSION TITLE
Can't click on input boxes with stylus

### DISCUSSION THREAD

**Discussion by jmalmsten** [wonderunit/storyboarder#2373] [General]
Just in the off-chance this project isn't as dead as it seems. 

I find it curious that I still am not able to click in GUI boxes with my wacom stylus. The tool-tips appear when I hover there, yes. But I cannot click checkboxes or number-input boxes and adjust them. I cannot click there and then type in numbers. I need to basically put the stylus down. Swap to my mouse and click in the box and then I switch to the keyboard to type in numbers. Then I need to switch back to the stylus or mouse to click play to test the timing out and then I repeat the procedure. For an app that promises to take away all the unnecessary bits. That seems a bit needlessly cumbersome. 

I would think that having a slider that can be accessed with the stylus to quickly adjust the board-length would be much preferred. Or something like that.

**Comment by lechouette** [comment]
> Just in the off-chance this project isn't as dead as it seems.
> 
> I find it curious that I still am not able to click in GUI boxes with my wacom stylus. The tool-tips appear when I hover there, yes. But I cannot click checkboxes or number-input boxes and adjust them. I cannot click there and then type in numbers. I need to basically put the stylus down. Swap to my mouse and click in the box and then I switch to the keyboard to type in numbers. Then I need to switch back to the stylus or mouse to click play to test the timing out and then I repeat the procedure. For an app that promises to take away all the unnecessary bits. That seems a bit needlessly cumbersome.
> 
> I would think that having a slider that can be accessed with the stylus to quickly adjust the board-length would be much preferred. Or something like that.

I had the same problem on an old Surface pro 2, had to click with the side button of the stylus and the tip, not just with the tip.
Not sure if this applies to more recent digitizers, hopefully it helps.
